evaluate_agent.py

```diff
@@ -0,0 +1,125 @@
+import os
+import sys
+import numpy as np
+import click
+from tqdm import tqdm
+from text_localization_environment import TextLocEnv
+from load_agent import load_agent
+from chainercv.evaluations.eval_detection_voc import eval_detection_voc
+
+
+@click.command()
+@click.option("--gpu", default=-1, help="ID of the GPU to be used. -1 if the CPU should be used instead.")
+@click.option("--imagefile", "-i", default='image_locations.txt', help="Path to the file containing the image locations.", type=click.Path(exists=True))
+@click.option("--boxfile", "-b", default='bounding_boxes.npy', help="Path to the bounding boxes.", type=click.Path(exists=True))
+@click.option("--agentdirectory", "-a", default='./agent', help="Path to the directory containing the agent that should be evaluated.", type=click.Path(exists=True))
+@click.option("--save/--dont-save", default=False, help="Boolean indicating whether the intermediate results of the evaluation should be saved.")
+def evaluate(gpu, imagefile, boxfile, agentdirectory, save):
+    max_steps_per_image = 200
+    max_steps_per_run = 40
```

Comment:
shouldn't max_steps_per_run = 50 considering that on train_agent.py the max_episode_len was 50?

Reply:
Good point – I'll change that. Thanks for noticing!